docs/quality.rst

```diff
@@ -30,48 +31,39 @@ The following descriptions of each quality code is taken from the WOCE manual.
 WOCE Bottle Quality Codes
 -------------------------
 
```

Comment:
The replaced format is a "definition list" when rendered in HTML and LaTeX (yay semantics). We could probably make it more clear that they are "Codes" in each definition term (e.g. "Code `1`"), but I'd want to keep it as a definition list rather than a table.